tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -248,7 +248,7 @@ Standard Section: "Comments" Comment*
 object TastyFormat {
 
   final val header: Array[Int] = Array(0x5C, 0xA1, 0xAB, 0x1F)
-  val MajorVersion: Int = 22
```

Comment:
Don't we need to pickle/unpickle TRANSPARENT?

Reply:
Done